src/fidesops/tasks/__init__.py

```diff
@@ -1,8 +1,17 @@
 from celery import Celery
 
+from fidesops.core.config import config
+
 app = Celery("tasks")
-app.config_from_object("fidesops.core.config", namespace="EXECUTION")
-app.autodiscover_tasks(["fidesops.tasks", "fidesops.tasks.scheduled"])
+app.conf.update(broker_url=config.execution.CELERY_BROKER_URL)
+app.conf.update(result_backend=config.execution.CELERY_RESULT_BACKEND)
+app.autodiscover_tasks(
+    [
+        "fidesops.tasks",
+        "fidesops.tasks.scheduled",
+        "fidesops.service.privacy_request.request_runner_service",
+    ]
+)
```

Comment:
have you managed to get this running locally outside of integration tests @seanpreston? I am not seeing any tasks being received or running at all.  I'm no longer getting 500's, but I don't think tasks are getting sent.  I'm not even sure that your run_privacy_task request is being registered:

```python3
>>> from celery import current_app
>>> current_app.tasks.keys()
dict_keys(['celery.chord', 'celery.accumulate', 'celery.chunks', 'celery.chord_unlock', 'celery.group', 'celery.map', 'celery.chain', 'celery.starmap', 'celery.backend_cleanup'])

```